Sources/Sentry/include/SentryHub+Private.h

```diff
@@ -53,6 +53,8 @@ SentryHub (Private)
                        withScope:(SentryScope *)scope
          additionalEnvelopeItems:(NSArray<SentryEnvelopeItem *> *)additionalEnvelopeItems;
 
+- (nullable SentrySession *)incrementSessionErrors;
+
```

Comment:
`m`: There should be no need to make this internally public.
```suggestion
```

Reply:
Ah right, not anymore after our block change.